src/app/app.component.ts

```diff
@@ -577,19 +598,20 @@ export class AppComponent implements OnInit, AfterViewInit, CanvasTableSelectLis
       this.selectedRowIds[this.selectedRowId] = true;
 
       if (!multiSelect && columnIndex > 1) {
+        let messageId: number;
         if (this.showingSearchResults) {
-          this.singlemailviewer.expectedMessageSize = this.searchService.api.getNumericValue(this.selectedRowId, 3);
```

Comment:
Confused here where expectedMessageSize went ?

Reply:
Hopefully described a bit better in https://github.com/runbox/runbox7/pull/612/commits/75616816347eb35ad40e5dc71479ba1053036c72 :)